backend/molgenis-emx2/src/main/java/org/molgenis/emx2/BinaryFileWrapper.java

```diff
@@ -11,6 +11,12 @@
   // these we calculate on construction
   String mimetype;
   String extension = "";
+  // this is used to skip file updates
+  boolean skip = false;
```

Comment:
In general i do not think its very nice to have this stateful bool, but i guess its kind of the java way, could it not be a function that calculates the bool response based on the state ( instead of a extra state flag)